<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="5" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="TestNGTestSuite" duration-ms="188" started-at="2019-08-02T13:10:11Z" finished-at="2019-08-02T13:10:11Z">
    <groups>
      <group name="test">
        <method signature="Groups.test1()[pri:0, instance:TestNG.Groups@96532d6]" name="test1" class="TestNG.Groups"/>
        <method signature="Groups.test2()[pri:0, instance:TestNG.Groups@96532d6]" name="test2" class="TestNG.Groups"/>
      </group> <!-- test -->
      <group name="logo">
        <method signature="Groups.printLogo1()[pri:0, instance:TestNG.Groups@96532d6]" name="printLogo1" class="TestNG.Groups"/>
        <method signature="Groups.printLogo2()[pri:0, instance:TestNG.Groups@96532d6]" name="printLogo2" class="TestNG.Groups"/>
      </group> <!-- logo -->
    </groups>
    <test name="TestNGTest" duration-ms="188" started-at="2019-08-02T13:10:11Z" finished-at="2019-08-02T13:10:11Z">
      <class name="TestNG.Groups">
        <test-method status="PASS" signature="printLogo1()[pri:0, instance:TestNG.Groups@96532d6]" name="printLogo1" duration-ms="19" started-at="2019-08-02T13:10:11Z" finished-at="2019-08-02T13:10:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- printLogo1 -->
        <test-method status="PASS" signature="printLogo2()[pri:0, instance:TestNG.Groups@96532d6]" name="printLogo2" duration-ms="0" started-at="2019-08-02T13:10:11Z" finished-at="2019-08-02T13:10:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- printLogo2 -->
        <test-method status="PASS" signature="test1()[pri:0, instance:TestNG.Groups@96532d6]" name="test1" duration-ms="1" started-at="2019-08-02T13:10:11Z" finished-at="2019-08-02T13:10:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method status="PASS" signature="test2()[pri:0, instance:TestNG.Groups@96532d6]" name="test2" duration-ms="0" started-at="2019-08-02T13:10:11Z" finished-at="2019-08-02T13:10:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
      </class> <!-- TestNG.Groups -->
      <class name="TestNG.DependsOnMethods">
        <test-method status="FAIL" signature="test1()[pri:0, instance:TestNG.DependsOnMethods@3796751b]" name="test1" duration-ms="1" started-at="2019-08-02T13:10:11Z" finished-at="2019-08-02T13:10:11Z">
          <exception class="java.lang.ArithmeticException">
            <message>
              <![CDATA[/ by zero]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArithmeticException: / by zero
at TestNG.DependsOnMethods.test1(DependsOnMethods.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArithmeticException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
      </class> <!-- TestNG.DependsOnMethods -->
    </test> <!-- TestNGTest -->
  </suite> <!-- TestNGTestSuite -->
</testng-results>
